MaybeGame/Player.cs

```diff
@@ -4,279 +4,86 @@ namespace MaybeGame
 {
     public class Player
     {
-        //TODO: Make a class to house stats and a class for Experience!!!! Make enum races
+        PlayerStats playerStats = new ();
```

Comment:
private? public?

Reply:
```suggestion
        private PlayerStats _playerStats = new ();
```

or

```suggestion
        public PlayerStats PlayerStats { get; set; } = new ();
```

Reply:
They are only accessible to player. So private